content/docs/manuals/unikernel-operations.md

```diff
@@ -1,12 +1,134 @@
+
+{{< callout context="tip" title="Looking for an example?" icon="outline/rocket" >}}
+Check out the **Unikraft Nginx** example in the Application Catalogue section of the wiki.
+{{< /callout >}}
+
+Once you've packaged your unikernel, you can upload it to a location accessible to Oakestra, such as a web server or a cloud storage service.
+
+## Creating a Service Deployment Descriptor for your Unikernel
+
+Unikernels can be deployed to Oakestra like any other service. You need to create a service deployment descriptor that describes the unikernel service you want to deploy.
+Here is an example of an Nginx server using Unikraft:
+
+```json {title="unikernel-nginx.json"}
+ {
+   "sla_version": "v2.0",
+   "customerID": "Admin",
+   "applications": [
+ {
+       "applicationID": "",
+       "application_name": "nginx",
+       "application_namespace": "test",
+       "application_desc": "Simple demo of a Unikraft Nginx server",
+       "microservices": [
+ {
+           "microserviceID": "",
+           "microservice_name": "nginx",
+           "microservice_namespace": "nginx",
+           "virtualization": "unikernel",
+           "cmd": [
+             ""
+ ],
+           "memory": 400,
+           "vcpus": 1,
+           "vgpus": 0,
+           "vtpus": 0,
+           "bandwidth_in": 0,
+           "bandwidth_out": 0,
+           "port": "9000:80",
+           "storage": 0,
+           "code": "https://github.com/oakestra/oakestra/releases/download/alpha-v0.4.301/nginx_amd64.tar.gz",
+           "arch": [
+             "amd64"
+ ],
+           "state": "",
+           "added_files": []
+ }
+ ]
+ }
+ ```
+
+ #### What's important to note in this SLA?
+
+- The `virtualization` field is set to `unikernel`
+- The `code` field contains the URL to the unikernel tarball. This URL must be accessible to the Oakestra worker nodes. We're using the Oakestra release page as an example here to host the tarball file.
+- The `arch` field specifies the architecture of the unikernel. In this case, it's `amd64`.
+
+The rest of the fields are similar to the ones used for container services.
+You can then deploy the application as usual using the Oakestra Dashboard, the Oakestra CLI, or the Oakestra APIs.
+
+{{< callout context="caution" title="Caution" icon="outline/alert-triangle" >}}
+You need at least one worker node in one of your clusters with a unikernel runtime enabled to deploy your unikernels. 
+{{< /callout >}}
+
+## Enable a KVM Runtime for an Oakestra Worker Node
+
+If your node supports nested virtualization and you have KVM installed, you can enable the KVM runtime for your Oakestra worker node.
+
+{{< callout context="tip" title="How to install KVM?" icon="outline/rocket" >}}
+Please follow [this](https://phoenixnap.com/kb/ubuntu-install-kvm) guide to install KVM on your worker node.
```

Comment:
I think we should mention that this is ubuntu specific

Reply:
```suggestion
E.g., If your worker node uses an Ubuntu distro, you can follow [this](https://phoenixnap.com/kb/ubuntu-install-kvm) guide to install KVM on your worker node.
```